clients/python/README.md

```diff
@@ -228,7 +228,7 @@ registered_model = registry.upload_artifact_and_register_model(
 First, you must ensure you are logged in the to appropriate OCI registry using
 `skopeo login`, `podman login`, or you can authenticate via the `OCIParams` by either:
 
-1. Reading a pull secret JSON from an environment variable. Default is `.dockerconfigjson`.
+1. Reading a pull secret JSON from an environment variable. Default is `DOCKER_CONFIG_JSON`.
```

Comment:
I am afraid that if some mechanism is loading Secret key/values as Env variables, it would end up as
`.dockerconfigjson` for Env variable

hence why it was like that in the first place :)

Reply:
I was going to ask if we could change it, but I guess there was a reason (our lint will even change `os.environ[".dockerconfigjson"]` to `os.environ[".DOCKERCONFIGJSON"]`. I don't love the way we're using a lowercase env var with a dot, but it's functional at least :smile: